### ISSUE TITLE
degraded sensitivity in positive mode [was: QTOF status 2018/09/05]

### ISSUE THREAD

**Issue by wdwvt1** [Stanford-ChEMH-MCAC/protocols#11]
As of 2018/09/05, the intensity of signals in positive mode on the QTOF appears to be somewhat degraded. Our best guess is a 2X decrease in signal intensity vs. 2 weeks ago. We are not seeing this same ionization drop in negative mode. 

A 5-10X drop in signal intensity was apparent earlier in the day, but at least part of that appears to have been solved by switching the ESI nebulizers. The nebulizer that was carrying sample has a metal piece lodged between the needle and nebulizer barrel as seen here. By switching that nebulizer with the gas injecting neublizer we restored some signal intensity (we think that is the reason at least...). 
![img_20180905_105109](https://user-images.githubusercontent.com/1048569/45125496-99f8ef00-b123-11e8-9d81-6bfcbac21992.jpg)

Our estimate of the signal intensity drop comes from C18 positive mode paraben standards run on 2018/08/10 and 2018/09/05. In blue are the old standards, in green the standards run today. The peaks are ~2X smaller in the standards run today.
![screen shot 2018-09-05 at 15 53 33](https://user-images.githubusercontent.com/1048569/45125592-feb44980-b123-11e8-98bf-e3f7a2eb2618.png)

**Comment by curt-f** [comment]
I just did our biweekly PM / cleaning.  More thorough washing of the needle housing (i.e. the "sheath" gas tube) removed more metal bits.  

The positive mode intensities have not recovered, but chromatography and mass accuracy is good.  I'll keep this open for now -- I doubt the signal intensity change is worthing getting too worked up about, but we should keep an eye on the situation to be sure.  (Reminder: comparing intensities between runs is a fraught exercise; please don't do it.)

![for_github](https://user-images.githubusercontent.com/36312929/45238838-05fc6400-b298-11e8-8625-cab2a68e0033.png)

**Comment by curt-f** [comment]
This still seems to be an issue.  Below is some data quantifying performance loss that Rob Phair sent from our own recent runs.

![loss_of_sensitivity_on_qtof](https://user-images.githubusercontent.com/36312929/45650551-761e9d00-ba83-11e8-9ce5-1dbfa1a8ad0d.png)

**Comment by curt-f** [comment]
The issue seems to appear in both (a) C18 and (b) HILIC modes.
 -- Seems unlikely to be buffer related

The issue is only in positive mode.  
-- Less likely that instrument (e.g. octopole or capillary) dirtiness is the problem, but I don't know enough to say for sure.

I'm on the phone with Agilent now.  I'd appreciate hearing from you about your recent runs to see if you've also experienced the problem.  [Update: no one home at Agilent; they will call me back and our service request / callback # is 8102105386.]

**Comment by curt-f** [comment]
From Jon:

https://mail.google.com/mail/u/0?ui=2&ik=d05d15b1e4&attid=0.1&permmsgid=msg-f:1611892485575745886&th=165e97aceadd215e&view=fimg&sz=s0-l75-ft&attbid=ANGjdJ__RqLrevG4C7mEiym7VfTfNvgdaP5JI2gRc9ophYEpkfdU1LtG77fcwJHKckBChcZkavDM-rtOcv7wIx3DefCuV-RKIl9neIAKMvrjxgVzg2zR3zgSMk-BN3I&disp=emb

https://mail.google.com/mail/u/0?ui=2&ik=d05d15b1e4&attid=0.2&permmsgid=msg-f:1611892485575745886&th=165e97aceadd215e&view=fimg&sz=s0-l75-ft&attbid=ANGjdJ8-3kbDM850AhcEa9UXJB1yeeu8m8TAC2a7hCMI10Cug8Lc3qvyzPz6VjzIEi3AyyUoEVyc3xV9UExzF0YgyH7RNIn-PB8a_KAH6vfA5nhJkLcODVs6vGuVNDc&disp=emb

**Comment by curt-f** [comment]
From Jon Long:

![image](https://user-images.githubusercontent.com/36312929/45652340-22af4d80-ba89-11e8-92fb-b69e21a114f1.png)

![image](https://user-images.githubusercontent.com/36312929/45652349-26db6b00-ba89-11e8-87cf-714584af5b1d.png)

**Comment by curt-f** [comment]
> On the advice of Agilent (and Jakub) I had to vent the instrument to I could clean the capillary.  Sensitivity was down for everyone across the board.
> 
> Pump down back to operating conditions will take overnight.  Tomorrow AM I'll try to adjust the schedule for impacted users.

**Comment by curt-f** [comment]
> The instrument is down.  After cleaning the capillary, on restart there appears to be a problem with the controller for turbopump 2.  Thus the instrument cannot maintain (or more likely, cannot verify) vacuum and cannot be used.  

> Agilent is on the case; stay tuned.

<img width="471" alt="screen shot 2018-09-19 at 09 51 46" src="https://user-images.githubusercontent.com/36312929/45768424-a8e6a380-bbf1-11e8-8ce5-fec26bd82e91.png">

**Comment by curt-f** [comment]
https://github.com/Stanford-ChEMH-MCAC/protocols/issues/13

**Comment by curt-f** [comment]
### TL;DR: 

Replacing the ion transfer capillary with a new one (Agilent part number G7604-60000) seems to have solved the issue. 

### Details

After Agilent fixed Turbopump #2, no one was able to calibrate or use the instrument despite it being at vacuum.  The sensitivity problem persisted.  Brand new calibrant, in positive and negative modes, and very low intensity (~5000 counts), and approximately 0 intensity for m/z's of less than 600-700 or so.

On the advice of multiple parties, (see above), I first tried to clean the ion transfer capillary (this did not work), and then on Monday evening I replaced it.  On Tuesday A.M. I did a full system tune in positive and negative modes (both the quadrupole and TOF), as well as transmission tunes in both pos and neg (in EDR 1700 mode) for the quad and the TOF.  The latter worked well.

Yesterday Will VT and I were able to run QC samples and this morning it seems that intensities are even higher than they have been historically.  

### Results of new QC runs

![screen shot 2018-10-03 at 10 42 24](https://user-images.githubusercontent.com/36312929/46428528-0a7e3600-c6f9-11e8-8c96-bf9fa2d0b823.png)
 Green: Negative mode QC from by WVT from yesterday.  Black: data from previous QC runs in October and September.

![screen shot 2018-10-03 at 10 43 34](https://user-images.githubusercontent.com/36312929/46428590-3c8f9800-c6f9-11e8-9f9d-2263f3b9c59f.png)
Orange: Positive mode QC run by WVT from yesterday.  Maroon/brown: data from previous QC runs in October and September

### Conclusions

The instrument is back and its better than ever!
